docker-cpp-demos/amazonlinux-docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Build docker with
+# docker build -t kinesis-video-producer-sdk-cpp-amazon-linux .
+#
+FROM amazonlinux:latest
+RUN yum update -y && \
+	yum upgrade -y && \
+	yum install -y \
+	autoconf \
+	automake  \
+	bison \
+	bzip2 \
+	cmake \
+	curl \
+	diffutils \
+	flex \
+	gcc \
+	gcc-c++ \
+	git \
+	gmp-devel \
+	gstreamer* \
+	libffi \
+	libffi-devel \
+	libmpc-devel \
+	libtool \
+	make \
+	m4 \
+	mpfr-devel \
+	pkgconfig \
+	vim \
+	wget \
+	xz
+
+RUN yum install -y libcurl-devel 
+
+RUN wget https://cmake.org/files/v3.12/cmake-3.12.3.tar.gz && \
```

Comment:
I'm not sure why we need to compile cmake from source since we already got the binary from `yum install`

Reply:
ouch, that's really weird.. haha